### DISCUSSION TITLE
Changelog only popup when you open Nebula

### DISCUSSION THREAD

**Discussion by dereineLauch** [cpiber/NebulaEnhance#14] [Ideas]
Currently, the Change log popup immediately after the update is installed. Most of the time this sucks because you're not using nebula at the time. So my suggestion is that there is an option that only when you open nebula.app the change log pops up.

**Comment by cpiber** [comment]
This is actually not hard to implement.

Personally I don't mind either way, but I guess if the extension updates in the background, the tab can be unexpected; I'll still open the tab "passive" though.

Not that any more updates are actually planned :) unless you have more feature requests
